angular.json

```diff
@@ -110,5 +110,8 @@
       }
     }
   },
-  "defaultProject": "quill"
+  "defaultProject": "quill",
+  "cli": {
+    "analytics": false
+  }
```

Comment:
Any reason why these three lines are added? If not please drop them

Reply:
They were created automatically, I didn't notice them removed on this PR